packages/backend/src/apps/postman-sms/actions/send-sms/index.ts

```diff
@@ -17,15 +18,20 @@ const action = {
   description: 'Sends an SMS under Gov.SG sender ID',
 
   /**
-   * CAVEAT
+   * FEATURE NOTE
    * ---
-   * THIS IS VERY LIKELY TO CHANGE! We currently only allow sending to the only
-   * template in Plumber's SMS campaign, using our API key.
+   * This is a simplified feature where we don't enable users to specify
+   * template variables (e.g. via a multi-row)
+   *
+   * This is because Postman does not provide an API to query template
+   * variables, nor template contents (which we can parse). It's more than
+   * likely that users will just get confused if we present them with a
+   * "Template Variables" multi-select.
    *
-   * But in the future, we want to allow users to provide their own API key and
-   * specify their own campaigns and templates. We can't do this right now
-   * because Postman does not provide an API to query templates / template
-   * variables.
+   * Instead, for this action, we will instruct users to set up a campaign whose
+   * template is just a {{body}} variable. We will provide a more advanced "Send
+   * SMS with Variables" action later on, for users who are comfortable working
+   * with template variables.
```

Comment:
question: this can only be added when postman provides the API right?

Reply:
yep yep, we can code it out for them tbh, but not a priority.